### PR TITLE
:sparkles: Add visualization and mouse control to paddings in frames with layout

### PR DESCRIPTION
**Pull request by pabloalba**
Implements https://tree.taiga.io/project/penpot/task/4839

### CODE REVIEW

File: frontend/src/app/main/ui/measurements.cljs [hunk 1]
```diff
@@ -261,3 +270,188 @@
...
+                          :padding padding
+                          :rect-data {:x x1
+                                      :y (if (:flip-y frame) (- y2 (:p1 padding)) y1)
+                                      :width width
+                                      :height (:p1 padding)
+                                      :initial-value (:p1 padding)
+                                      :resize-type (if (:flip-y frame) :bottom :top)
+                                      :resize-axis :y
+                                      :negate? (:p1 negate)
+                                      :mouse-enter (partial mouse-enter :p1)
```
Comment (Alotor) [inline frontend/src/app/main/ui/measurements.cljs]: Why put some of this properties inside "rect-data"? Seems more logical to put them directly in the props. For example: mouse-enter, mouse-leave...

File: frontend/src/app/main/ui/measurements.cljs [hunk 2]
```diff
@@ -261,3 +270,188 @@
...
+                :p2 (if (:flip-x frame) true false)
+                :p3 (if (:flip-y frame) true false)
+                :p4 (if (:flip-x frame) true false)}
+
+        negate (cond-> negate
+                       (= :fix (:layout-item-h-sizing frame)) (assoc :p2 (not (:p2 negate)))
+                       (= :fix (:layout-item-v-sizing frame)) (assoc :p3 (not (:p3 negate))))]
+
+    [:g.paddings {:pointer-events "visible"}
+     [:& padding-display {:frame-id frame-id
```
Comment (Alotor) [inline frontend/src/app/main/ui/measurements.cljs]: Maybe you could create a `for` loop for these?

File: frontend/src/app/main/ui/measurements.cljs (hunk 1 above)
Comment (pabloalba) [reply frontend/src/app/main/ui/measurements.cljs]: Ok, I've extracted them

File: frontend/src/app/main/ui/measurements.cljs (hunk 2 above)
Comment (pabloalba) [reply frontend/src/app/main/ui/measurements.cljs]: Done

**Pull request merged**